core/src/main/resources/hudson/model/Job/buildTimeTrend_resources.js

```diff
@@ -122,3 +122,82 @@ function generateSVGIcon(iconName, iconSizeClass) {
 
   return span;
 }
+
+/**
+ * Public method to be called by progressiveRendering's callback
+ */
+window.displayBuilds = function (data) {
+  var rootUrl = document.head.getAttribute("data-rooturl");
+  var p = document.getElementById("projectStatus");
+  p.style.display = "";
+  for (var x = 0; data.length > x; x++) {
+    var e = data[x];
+    var tr = document.createElement("tr");
+
+    var td1 = document.createElement("td");
+    td1.setAttribute("data", e.iconColorOrdinal);
+    td1.classList.add("jenkins-table__cell--tight", "jenkins-table__icon");
+    var div1 = document.createElement("div");
+    div1.classList.add("jenkins-table__cell__button-wrapper");
+    var svg = generateSVGIcon(e.iconName, p.dataset.iconSizeClass);
+    div1.appendChild(svg);
+    td1.appendChild(div1);
+    tr.appendChild(td1);
+
+    var td2 = document.createElement("td");
+    var a1 = document.createElement("a");
+    a1.classList.add("jenkins-table__link", "model-link");
+    a1.href = rootUrl + "/" + e.parentUrl;
+    var span1 = document.createElement("span");
+    span1.textContent = e.parentFullDisplayName;
```

Comment:
For some reason, this value (and only this one) is pre-escaped in Java, resulting in double-escaping with a side of `<wbr>`:

https://github.com/jenkinsci/jenkins/blob/ccba9764a7d75372910708b68fc2d83ade5f6bcb/core/src/main/java/jenkins/widgets/BuildListTable.java#L46

<img width="962" alt="Screenshot 2023-04-17 at 09 22 11" src="https://user-images.githubusercontent.com/1831569/232413467-80c01c77-90dd-44ab-a9d3-30e5f581d133.png">

Reply:
commit e2eddebc03